daemon/containerd/image.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/docker/docker/image"
 	"github.com/docker/docker/layer"
 	"github.com/docker/go-connections/nat"
+	"github.com/moby/buildkit/frontend/dockerfile/dockerfile2llb"
```

Comment:
I'm still not comfortable with having our internal types defined by another project; see the discussion on https://github.com/rumpl/moby/pull/121#discussion_r1044446599

If that type is implementing the image "v1" spec defined in this repository (https://github.com/moby/moby/tree/master/image/spec), and we don't have a type for that in this repository, we should define it here

Reply:
I removed the commit